src/xenia/ui/ui_event.h

```diff
@@ -40,11 +40,107 @@ class FileDropEvent : public UIEvent {
 
 class KeyEvent : public UIEvent {
  public:
-  KeyEvent(Window* target, int key_code, int repeat_count, bool prev_state,
-           bool modifier_shift_pressed, bool modifier_ctrl_pressed,
-           bool modifier_alt_pressed, bool modifier_super_pressed)
+  // TODO(dougvj) The list of keycodes is probably incomplete but may be
+  // sufficient. Maybe there is a better way?
+  enum class Key {
```

Comment:
It's preferable that we normalize keys to Windows virtual key codes, as those are sent to the game via XamInputGetKeystroke. This would also allow us to skip most of the translation for Windows host, maybe except for some fixups if needed to match the Xbox 360 controller keyboard better.